Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/icx-d-test-nic-simple
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (set args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/conf/change_args.py '{"dpdk_dir":"/opt/APP/utility/DPDK/dpdk-21.11/","sut_conf":{"board_type":"rp","host":"10.239.115.71","pf1":"enp137s0f0","os":"UBUNTU","PF1_NIC_type":"nic-in-soc"},"pkg_generator":{"host":"10.239.115.16","pf1":"ens801f0"},"drive_info":{"drive_version":"1.7.19"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics -s -m nicSimple --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_simple_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 17 items / 9 deselected / 8 selected

npxtester/Tests/testNics/test_mtu.py 2022-03-26 01:38:21 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Jumbo_Packet_CVL_01_38_21/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Jumbo_Packet_CVL_01_38_21
2022-03-26 01:38:21 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-03-26 01:38:21 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-03-26 01:38:21 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 01:38:21 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:38:21 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 01:38:21 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 01:38:21 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:38:21 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 01:38:21 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 01:38:24 - sshInstance.py - [line:50] - root - INFO - modinfo ice | grep ^version:
2022-03-26 01:38:24 - sshInstance.py - [line:54] - root - INFO - version:        1.7.19

2022-03-26 01:38:32 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 01:38:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:38:43 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 01:38:47 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:38:55 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0
2022-03-26 01:38:55 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.7.19
firmware-version: 2.19 0x8000bafe 1.3083.0
expansion-rom-version: 
bus-info: 0000:89:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-03-26 01:39:03 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 mtu 1500
2022-03-26 01:39:03 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:39:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 mtu 1500
2022-03-26 01:39:11 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:39:19 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 01:39:19 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 14  bytes 2451 (2.4 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:39:28 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Mar 25 07:55:18 2022 from 10.239.182.99
[root@redhat8 ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.4[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | What is dead may never die![0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   |                     -- Python 2[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-03-26 01:39:28 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:39:35 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="00:00:00:00:33:00")/IP(len=46)/Raw(load='X'*14480)],iface="ens801f0")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-03-26 01:39:35 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:39:38 - test_mtu.py - [line:92] - root - INFO - success to send package to network
2022-03-26 01:39:42 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="00:00:00:00:33:00")/IP(len=46)/Raw(load='X'*15500)],iface="ens801f0")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/sendrecv.py", line 379, in sendp
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/sendrecv.py", line 308, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/arch/linux.py", line 476, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/supersocket.py", line 76, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-03-26 01:39:42 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:39:45 - test_mtu.py - [line:102] - root - INFO - result is right
.2022-03-26 01:39:48 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 mtu 1500
2022-03-26 01:39:48 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:39:56 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 mtu 1500
2022-03-26 01:39:56 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:40:04 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 01:40:04 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:40:13 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 01:40:13 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:40:18 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 01:40:18 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Jumbo_Packet_CVL_01_38_21
2022-03-26 01:40:21 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 01:40:21 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:40:26 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 01:40:26 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-03-26 01:40:26 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-03-26 01:40:26 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_MTU_set_CVL_01_40_26/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_MTU_set_CVL_01_40_26
2022-03-26 01:40:26 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-03-26 01:40:26 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-03-26 01:40:26 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 01:40:26 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:40:26 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 01:40:26 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 01:40:26 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:40:26 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 01:40:26 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 01:40:29 - sshInstance.py - [line:50] - root - INFO - modinfo ice | grep ^version:
2022-03-26 01:40:29 - sshInstance.py - [line:54] - root - INFO - version:        1.7.19

2022-03-26 01:40:37 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 01:40:40 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:40:48 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 01:40:52 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:41:00 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0
2022-03-26 01:41:00 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.7.19
firmware-version: 2.19 0x8000bafe 1.3083.0
expansion-rom-version: 
bus-info: 0000:89:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-03-26 01:41:08 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 mtu 1500
2022-03-26 01:41:08 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:41:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 mtu 2000
2022-03-26 01:41:16 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:41:24 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 01:41:24 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 14  bytes 2451 (2.4 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:41:33 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Sat Mar 26 01:24:59 2022 from 10.239.182.99
[root@redhat8 ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.4[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | To craft a packet, you have to be a[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   | packet, and learn how to swim in[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   | the wires and in the waves.[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m[34m[1m   |        -- Jean-Claude Van Damme[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m[34m[1m   |[0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-03-26 01:41:33 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:41:40 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="00:00:00:00:33:00")/IP(len=46)/Raw(load='X'*18890)],iface="ens801f0")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-03-26 01:41:40 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:41:43 - test_mtu.py - [line:132] - root - INFO - success to send package to network
2022-03-26 01:41:47 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="00:00:00:00:33:00")/IP(len=46)/Raw(load='X'*20010)],iface="ens801f0")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/sendrecv.py", line 379, in sendp
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/sendrecv.py", line 308, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/arch/linux.py", line 476, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy-2.4.4-py3.6.egg/scapy/supersocket.py", line 76, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-03-26 01:41:47 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:41:50 - test_mtu.py - [line:142] - root - INFO - result is right
.2022-03-26 01:41:53 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 mtu 1500
2022-03-26 01:41:53 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 mtu 1500
2022-03-26 01:42:01 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:09 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 01:42:09 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:17 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 01:42:17 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:22 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 01:42:22 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_PI_Networking_MTU_set_CVL_01_40_26
2022-03-26 01:42:25 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 01:42:25 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:30 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 01:42:30 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-03-26 01:42:30 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
                                  [ 25%]
npxtester/Tests/testNics/test_nics.py 2022-03-26 01:42:30 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-03-26 01:42:30 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_silver_Networking_ConnectivityEnableDisable_01_42_30/test_silver_Networking_ConnectivityEnableDisable.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_silver_Networking_ConnectivityEnableDisable_01_42_30
2022-03-26 01:42:30 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-03-26 01:42:30 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 01:42:30 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 01:42:30 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:42:31 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 01:42:31 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 01:42:31 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:42:31 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 01:42:31 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 01:42:31 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 01:42:34 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 01:42:34 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 01:42:39 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 01:42:39 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 01:42:42 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 01:42:45 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:42:50 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 01:42:50 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 01:42:53 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 01:42:57 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:43:02 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 01:43:05 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 01:43:05 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:43:13 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 01:43:13 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 01:43:21 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 01:43:21 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:43:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 01:43:29 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946356928  bytes 63492550086574 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559615  bytes 161885741968 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:43:37 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:43:37 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:43:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11
2022-03-26 01:43:45 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:43:54 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 01:44:27 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.021 ms
2022-03-26 01:43:55 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.018 ms
2022-03-26 01:43:56 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.012 ms
2022-03-26 01:43:57 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.011 ms
2022-03-26 01:43:58 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4100ms
rtt min/avg/max/mdev = 0.011/0.014/0.021/0.004 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:43:58 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:07 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:07 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 down
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:44:16 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:16 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 up
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:44:22 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:44:22 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:44:31 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.023 ms
2022-03-26 01:44:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.021 ms
2022-03-26 01:44:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.016 ms
2022-03-26 01:44:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.016 ms
2022-03-26 01:44:35 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.015 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4101ms
rtt min/avg/max/mdev = 0.015/0.018/0.023/0.003 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:44:35 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:44 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:44 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 down
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:44:53 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:44:53 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 up
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:44:59 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:44:59 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:45:08 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.021 ms
2022-03-26 01:45:09 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.016 ms
2022-03-26 01:45:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.013 ms
2022-03-26 01:45:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.012 ms
2022-03-26 01:45:12 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.013 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.012/0.015/0.021/0.003 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:45:12 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:45:21 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:45:21 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 down
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:45:30 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:45:30 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 up
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:45:36 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:45:36 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:45:45 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.022 ms
2022-03-26 01:45:46 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.017 ms
2022-03-26 01:45:47 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.006 ms
2022-03-26 01:45:48 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.012 ms
2022-03-26 01:45:49 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.012 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4075ms
rtt min/avg/max/mdev = 0.006/0.013/0.022/0.005 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:45:49 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:45:58 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:45:58 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 down
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:46:07 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:46:07 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 up
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:46:13 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:46:13 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:46:22 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.023 ms
2022-03-26 01:46:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.015 ms
2022-03-26 01:46:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.012 ms
2022-03-26 01:46:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.011 ms
2022-03-26 01:46:26 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4077ms
rtt min/avg/max/mdev = 0.011/0.014/0.023/0.004 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:46:26 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:46:35 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:46:35 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 down
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:46:44 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:46:44 - sshInstance.py - [line:100] - root - INFO - ifconfig enp137s0f0 up
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:46:50 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:46:50 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:46:59 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.023 ms
2022-03-26 01:47:00 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.014 ms
2022-03-26 01:47:01 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.012 ms
2022-03-26 01:47:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.011 ms
2022-03-26 01:47:03 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4079ms
rtt min/avg/max/mdev = 0.011/0.014/0.023/0.004 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:47:03 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
.2022-03-26 01:47:09 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 01:47:09 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:17 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 01:47:17 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:25 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 01:47:25 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 01:47:34 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:42 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 01:47:42 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:47 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 01:47:47 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_silver_Networking_ConnectivityEnableDisable_01_42_30
2022-03-26 01:47:50 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 01:47:50 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:47:55 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 01:47:55 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 01:47:55 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 01:47:55 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-03-26 01:47:55 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_silver_ConnectivityReboot_01_47_55/test_silver_ConnectivityReboot.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_silver_ConnectivityReboot_01_47_55
2022-03-26 01:47:55 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_ConnectivityReboot case ============================
2022-03-26 01:47:55 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 01:47:55 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 01:47:55 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:47:55 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 01:47:55 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 01:47:55 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:47:55 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 01:47:55 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 01:47:55 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 01:47:58 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 01:47:58 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 01:48:03 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 01:48:03 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 01:48:06 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 01:48:09 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:48:14 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 01:48:14 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 01:48:17 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 01:48:21 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:48:26 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 01:48:29 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 01:48:29 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:48:37 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 01:48:37 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 01:48:45 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 01:48:45 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:48:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 01:48:53 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357013  bytes 63492550102060 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559628  bytes 161885744546 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 01:49:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11
2022-03-26 01:49:01 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:49:09 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:49:09 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:49:14 - test_nics.py - [line:121] - root - INFO - -- reboot 1 times --
2022-03-26 01:49:18 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 01:49:52 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.145 ms
2022-03-26 01:49:19 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.043 ms
2022-03-26 01:49:20 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.024 ms
2022-03-26 01:49:21 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.032 ms
2022-03-26 01:49:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.032 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4072ms
rtt min/avg/max/mdev = 0.024/0.055/0.145/0.045 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:49:22 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:49:29 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.153 ms
2022-03-26 01:49:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.040 ms
2022-03-26 01:49:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.029 ms
2022-03-26 01:49:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.030 ms
2022-03-26 01:49:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.029 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.029/0.056/0.153/0.048 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:49:33 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:49:42 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:49:42 - sshInstance.py - [line:100] - root - INFO - reboot

2022-03-26 01:49:45 - sshInstance.py - [line:160] - root - INFO - sut server reboot, please wait ......
2022-03-26 01:50:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4130ms
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:50:50 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:50:50 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:50:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4079ms
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:00 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4093ms
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:10 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:15 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4107ms
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:20 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4122ms
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:30 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4073ms
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:41 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:46 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4088ms
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:51 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:51:51 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:51:56 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4102ms
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:01 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:01 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:06 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4118ms
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:11 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:11 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:16 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4133ms
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:21 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:21 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:26 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4084ms
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:31 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:31 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:36 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4100ms
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:41 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:41 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:46 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=1.78 ms
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 1 received, 80% packet loss, time 4115ms
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 1.779/1.779/1.779/0.000 ms
2022-03-26 01:52:51 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:52:51 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:52:56 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.817 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.785 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.929 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.829 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.793 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4066ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.785/0.830/0.929/0.060 ms
2022-03-26 01:53:01 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:53:01 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:53:01 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 01:56:01 - test_nics.py - [line:121] - root - INFO - -- reboot 2 times --
2022-03-26 01:56:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 01:56:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 01:56:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 01:56:04 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 01:56:04 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:56:12 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 01:56:12 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 01:56:21 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 01:51:11 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.103 ms
2022-03-26 01:56:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.033 ms
2022-03-26 01:56:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.035 ms
2022-03-26 01:56:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.033 ms
2022-03-26 01:56:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.033 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4089ms
rtt min/avg/max/mdev = 0.033/0.047/0.103/0.027 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:56:25 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:56:32 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.142 ms
2022-03-26 01:56:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.034 ms
2022-03-26 01:56:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.033 ms
2022-03-26 01:56:35 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.032 ms
2022-03-26 01:56:36 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.030 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4083ms
rtt min/avg/max/mdev = 0.030/0.054/0.142/0.043 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 01:56:36 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:56:45 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 01:56:45 - sshInstance.py - [line:100] - root - INFO - reboot

2022-03-26 01:56:48 - sshInstance.py - [line:160] - root - INFO - sut server reboot, please wait ......
2022-03-26 01:57:48 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4131ms
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:57:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:57:53 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:57:58 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4082ms
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:03 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:08 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4096ms
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:13 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:13 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:18 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4110ms
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:23 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:23 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:28 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4124ms
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:33 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:33 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:38 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4074ms
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:43 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:43 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:48 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4089ms
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:53 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:58:53 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:58:58 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4103ms
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:03 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:03 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:09 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4116ms
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:14 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:14 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:19 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4130ms
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:24 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:24 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:29 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4080ms
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:34 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:34 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:39 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4092ms
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:44 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:44 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:49 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=1.74 ms
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.827 ms
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 2 received, 60% packet loss, time 4084ms
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.827/1.283/1.739/0.456 ms
2022-03-26 01:59:54 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 01:59:54 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 01:59:59 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.864 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.937 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.756 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.741 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.924 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4054ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.741/0.844/0.937/0.086 ms
2022-03-26 02:00:04 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:00:04 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:00:04 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:03:04 - test_nics.py - [line:121] - root - INFO - -- reboot 3 times --
2022-03-26 02:03:04 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:03:04 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:03:04 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:03:07 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:03:07 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:03:15 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:03:15 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:03:24 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 01:58:14 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.100 ms
2022-03-26 02:03:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.044 ms
2022-03-26 02:03:26 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.037 ms
2022-03-26 02:03:27 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.036 ms
2022-03-26 02:03:28 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.038 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4080ms
rtt min/avg/max/mdev = 0.036/0.051/0.100/0.024 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:03:28 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:03:35 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.159 ms
2022-03-26 02:03:36 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.054 ms
2022-03-26 02:03:37 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.035 ms
2022-03-26 02:03:38 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.032 ms
2022-03-26 02:03:39 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.031 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4101ms
rtt min/avg/max/mdev = 0.031/0.062/0.159/0.049 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:03:39 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:03:48 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:03:48 - sshInstance.py - [line:100] - root - INFO - reboot

2022-03-26 02:03:51 - sshInstance.py - [line:160] - root - INFO - sut server reboot, please wait ......
2022-03-26 02:04:51 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4129ms
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:04:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:04:56 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:01 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4077ms
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:06 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:11 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4089ms
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:16 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:21 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4105ms
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:26 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:31 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4118ms
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:36 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:36 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:41 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4133ms
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:46 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:46 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:05:51 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4083ms
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:56 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:05:56 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:01 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4097ms
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:06 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:11 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4110ms
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:16 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:16 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:21 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4055ms
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:26 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:26 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:31 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4074ms
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:37 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:37 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:42 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4087ms
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:47 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:47 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:06:52 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=1.36 ms
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 1 received, 80% packet loss, time 4102ms
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 1.357/1.357/1.357/0.000 ms
2022-03-26 02:06:57 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:06:57 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:07:02 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.732 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.816 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.915 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.761 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.703 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4051ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.703/0.785/0.915/0.078 ms
2022-03-26 02:07:07 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:07:07 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:07:07 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:10:07 - test_nics.py - [line:121] - root - INFO - -- reboot 4 times --
2022-03-26 02:10:07 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:10:07 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:10:07 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:10:10 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:10:10 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:10:18 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:10:18 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:10:27 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 02:05:17 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.084 ms
2022-03-26 02:10:28 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.048 ms
2022-03-26 02:10:29 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.035 ms
2022-03-26 02:10:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.033 ms
2022-03-26 02:10:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.032 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4079ms
rtt min/avg/max/mdev = 0.032/0.046/0.084/0.019 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:10:31 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:10:38 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.125 ms
2022-03-26 02:10:39 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.031 ms
2022-03-26 02:10:40 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.034 ms
2022-03-26 02:10:41 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.029 ms
2022-03-26 02:10:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.030 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4073ms
rtt min/avg/max/mdev = 0.029/0.049/0.125/0.037 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:10:42 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:10:51 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:10:51 - sshInstance.py - [line:100] - root - INFO - reboot

2022-03-26 02:10:54 - sshInstance.py - [line:160] - root - INFO - sut server reboot, please wait ......
2022-03-26 02:11:54 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4105ms
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:11:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:11:59 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:04 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4114ms
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:09 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:14 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4127ms
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:19 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:24 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4077ms
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:29 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:34 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4092ms
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:39 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:44 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4107ms
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:49 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:12:54 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4120ms
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:59 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:12:59 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:04 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4135ms
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:09 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:09 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:14 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4085ms
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:19 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:19 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:24 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4097ms
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:29 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:29 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:34 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4111ms
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:39 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:39 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:44 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4126ms
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:49 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:13:49 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:13:54 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=1.91 ms
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 1 received, 80% packet loss, time 4076ms
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 1.908/1.908/1.908/0.000 ms
2022-03-26 02:14:00 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:14:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:14:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.908 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.878 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=1.02 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.710 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.679 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4090ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.679/0.838/1.018/0.130 ms
2022-03-26 02:14:10 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:14:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:14:10 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:17:10 - test_nics.py - [line:121] - root - INFO - -- reboot 5 times --
2022-03-26 02:17:10 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:17:10 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:17:10 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:17:13 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:17:13 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:17:21 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:17:21 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:17:30 - sshInstance.py - [line:75] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

Last login: Sat Mar 26 02:12:20 2022 from 10.239.182.99
root@npx-sut3-ice-d-auto:~# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.095 ms
2022-03-26 02:17:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.045 ms
2022-03-26 02:17:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.031 ms
2022-03-26 02:17:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.036 ms
2022-03-26 02:17:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.036 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4082ms
rtt min/avg/max/mdev = 0.031/0.048/0.095/0.023 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:17:34 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:17:41 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.146 ms
2022-03-26 02:17:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.037 ms
2022-03-26 02:17:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.030 ms
2022-03-26 02:17:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.030 ms
2022-03-26 02:17:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.026 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4075ms
rtt min/avg/max/mdev = 0.026/0.053/0.146/0.046 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-26 02:17:45 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:17:54 - sshInstance.py - [line:98] - root - INFO - --cmd is end--
2022-03-26 02:17:54 - sshInstance.py - [line:100] - root - INFO - reboot

2022-03-26 02:17:57 - sshInstance.py - [line:160] - root - INFO - sut server reboot, please wait ......
2022-03-26 02:18:57 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4092ms
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:02 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:07 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4105ms
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:12 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:17 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4120ms
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:22 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:27 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4134ms
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:32 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:37 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4083ms
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:42 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:47 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4097ms
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:19:52 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:19:57 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4111ms
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:02 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:07 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4126ms
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:12 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:17 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4076ms
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:22 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:22 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:27 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4090ms
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:32 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:32 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:37 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4105ms
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:42 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:42 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:47 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 4119ms
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:52 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:20:52 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:20:57 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=1.78 ms
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 1 received, 80% packet loss, time 4133ms
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 1.780/1.780/1.780/0.000 ms
2022-03-26 02:21:02 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:21:02 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:21:07 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.723 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.761 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.750 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.778 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.623 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4083ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.623/0.727/0.778/0.054 ms
2022-03-26 02:21:12 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:21:12 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:21:12 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
.2022-03-26 02:24:16 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:24:16 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:24:16 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:24:16 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:24:16 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:24:19 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:24:19 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:24:27 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 02:24:27 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:24:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 02:24:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:24:43 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 02:24:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:24:48 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 02:24:48 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_silver_ConnectivityReboot_01_47_55
2022-03-26 02:24:51 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 02:24:51 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:24:56 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 02:24:56 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:24:56 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 02:24:56 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_ConnectivityReboot case ============================
2022-03-26 02:24:56 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_gold_PI_Networking_SRIOV_CheckDevice_02_24_56/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_gold_PI_Networking_SRIOV_CheckDevice_02_24_56
2022-03-26 02:24:56 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-03-26 02:24:56 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 02:24:56 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:24:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:24:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:24:56 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 02:24:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:24:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 02:24:56 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:24:56 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 02:24:59 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 02:24:59 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 02:25:04 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 02:25:04 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 02:25:07 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 02:25:10 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:25:15 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 02:25:15 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 02:25:18 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 02:25:23 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:25:28 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 02:25:31 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:25:31 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:25:39 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 02:25:39 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 02:25:47 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 02:25:47 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:25:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 02:25:55 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357344  bytes 63492550176324 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559711  bytes 161885753889 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:26:03 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:26:03 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11

2022-03-26 02:26:11 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:19 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v0 192.168.10.11
2022-03-26 02:26:19 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:27 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v1 192.168.10.12
2022-03-26 02:26:27 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:35 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v2 192.168.10.13
2022-03-26 02:26:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:43 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v3 192.168.10.14
2022-03-26 02:26:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:26:51 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11
2022-03-26 02:26:55 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.025 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.019 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.011 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.011 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4082ms
rtt min/avg/max/mdev = 0.011/0.015/0.025/0.005 ms

2022-03-26 02:27:03 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v0 up
2022-03-26 02:27:03 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:27:11 - sshInstance.py - [line:50] - root - INFO - nmcli device connect enp137s0f0v0
2022-03-26 02:27:11 - sshInstance.py - [line:54] - root - INFO - bash: nmcli: command not found

2022-03-26 02:27:19 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I enp137s0f0v0 fe80::3efd:feff:fed2:73e0
2022-03-26 02:27:23 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than: enp137s0f0v0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0v0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v0: icmp_seq=1 ttl=64 time=0.120 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v0: icmp_seq=2 ttl=64 time=0.039 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v0: icmp_seq=3 ttl=64 time=0.031 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v0: icmp_seq=4 ttl=64 time=0.031 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v0: icmp_seq=5 ttl=64 time=0.029 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4077ms
rtt min/avg/max/mdev = 0.029/0.050/0.120/0.035 ms

2022-03-26 02:27:31 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11
2022-03-26 02:27:35 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.018 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.016 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.012 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.011 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4078ms
rtt min/avg/max/mdev = 0.011/0.013/0.018/0.003 ms

2022-03-26 02:27:43 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v1 up
2022-03-26 02:27:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:27:51 - sshInstance.py - [line:50] - root - INFO - nmcli device connect enp137s0f0v1
2022-03-26 02:27:51 - sshInstance.py - [line:54] - root - INFO - bash: nmcli: command not found

2022-03-26 02:27:59 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I enp137s0f0v1 fe80::3efd:feff:fed2:73e0
2022-03-26 02:28:03 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than: enp137s0f0v1
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0v1: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v1: icmp_seq=1 ttl=64 time=0.102 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v1: icmp_seq=2 ttl=64 time=0.039 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v1: icmp_seq=3 ttl=64 time=0.032 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v1: icmp_seq=4 ttl=64 time=0.036 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v1: icmp_seq=5 ttl=64 time=0.030 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4076ms
rtt min/avg/max/mdev = 0.030/0.047/0.102/0.027 ms

2022-03-26 02:28:11 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11
2022-03-26 02:28:15 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.018 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.016 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.012 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.008 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4077ms
rtt min/avg/max/mdev = 0.008/0.013/0.018/0.003 ms

2022-03-26 02:28:23 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v2 up
2022-03-26 02:28:24 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:28:32 - sshInstance.py - [line:50] - root - INFO - nmcli device connect enp137s0f0v2
2022-03-26 02:28:32 - sshInstance.py - [line:54] - root - INFO - bash: nmcli: command not found

2022-03-26 02:28:40 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I enp137s0f0v2 fe80::3efd:feff:fed2:73e0
2022-03-26 02:28:44 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than: enp137s0f0v2
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0v2: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v2: icmp_seq=1 ttl=64 time=0.130 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v2: icmp_seq=2 ttl=64 time=0.039 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v2: icmp_seq=3 ttl=64 time=0.027 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v2: icmp_seq=4 ttl=64 time=0.030 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v2: icmp_seq=5 ttl=64 time=0.033 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4076ms
rtt min/avg/max/mdev = 0.027/0.051/0.130/0.039 ms

2022-03-26 02:28:52 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11
2022-03-26 02:28:56 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.018 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.013 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.013 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.011 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.011 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4079ms
rtt min/avg/max/mdev = 0.011/0.013/0.018/0.002 ms

2022-03-26 02:29:04 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0v3 up
2022-03-26 02:29:04 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:29:12 - sshInstance.py - [line:50] - root - INFO - nmcli device connect enp137s0f0v3
2022-03-26 02:29:12 - sshInstance.py - [line:54] - root - INFO - bash: nmcli: command not found

2022-03-26 02:29:20 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I enp137s0f0v3 fe80::3efd:feff:fed2:73e0
2022-03-26 02:29:24 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than: enp137s0f0v3
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0v3: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v3: icmp_seq=1 ttl=64 time=0.117 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v3: icmp_seq=2 ttl=64 time=0.033 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v3: icmp_seq=3 ttl=64 time=0.028 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v3: icmp_seq=4 ttl=64 time=0.030 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0v3: icmp_seq=5 ttl=64 time=0.025 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4078ms
rtt min/avg/max/mdev = 0.025/0.046/0.117/0.035 ms

.2022-03-26 02:29:32 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:29:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:29:43 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:29:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:29:51 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 02:29:51 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:29:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 02:29:59 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:30:07 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 02:30:07 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:30:12 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 02:30:12 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_gold_PI_Networking_SRIOV_CheckDevice_02_24_56
2022-03-26 02:30:15 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 02:30:15 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:30:20 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 02:30:20 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:30:20 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 02:30:20 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-03-26 02:30:20 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_InternetProtocolv4v6_02_30_20/test_PI_Networking_InternetProtocolv4v6.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_InternetProtocolv4v6_02_30_20
2022-03-26 02:30:20 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-03-26 02:30:20 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 02:30:20 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:30:20 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:30:20 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:30:20 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 02:30:20 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:30:20 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 02:30:20 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:30:20 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 02:30:23 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 02:30:23 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 02:30:28 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 02:30:28 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 02:30:31 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 02:30:34 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:30:39 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 02:30:39 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 02:30:42 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 02:30:46 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:30:51 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 02:30:54 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:30:54 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:31:02 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 02:31:02 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 02:31:10 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 02:31:10 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:31:18 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 02:31:18 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357442  bytes 63492550188455 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559753  bytes 161885759585 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:31:26 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:31:26 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:31:34 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11
2022-03-26 02:31:34 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:31:42 - sshInstance.py - [line:50] - root - INFO - ping -w 5 192.168.10.11
2022-03-26 02:31:47 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.105 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.060 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.036 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.036 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.035 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.035/0.054/0.105/0.027 ms

2022-03-26 02:31:55 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I enp137s0f0 fe80::3efd:feff:fed2:73e0
2022-03-26 02:32:00 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than: enp137s0f0
PING fe80::3efd:feff:fed2:73e0(fe80::3efd:feff:fed2:73e0) from :: enp137s0f0: 56 data bytes
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=1 ttl=64 time=0.141 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=2 ttl=64 time=0.040 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=3 ttl=64 time=0.032 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=4 ttl=64 time=0.040 ms
64 bytes from fe80::3efd:feff:fed2:73e0%enp137s0f0: icmp_seq=5 ttl=64 time=0.030 ms

--- fe80::3efd:feff:fed2:73e0 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4103ms
rtt min/avg/max/mdev = 0.030/0.056/0.141/0.042 ms

.2022-03-26 02:32:08 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:32:08 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:16 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:32:16 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:24 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 02:32:24 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:32 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 02:32:32 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:40 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 02:32:40 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:45 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 02:32:45 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_PI_Networking_InternetProtocolv4v6_02_30_20
2022-03-26 02:32:48 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 02:32:48 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:32:53 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 02:32:53 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:32:53 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 02:32:53 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-03-26 02:32:53 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Link_State_Check_02_32_53/test_PI_Networking_Link_State_Check.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Link_State_Check_02_32_53
2022-03-26 02:32:53 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Link_State_Check case ============================
2022-03-26 02:32:53 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 02:32:53 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:32:53 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:32:53 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:32:53 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 02:32:53 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:32:53 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 02:32:53 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:32:53 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 02:32:56 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 02:32:56 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 02:33:01 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 02:33:01 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 02:33:04 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 02:33:07 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:33:12 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 02:33:12 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 02:33:15 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 02:33:19 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:33:24 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 02:33:27 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:33:27 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:33:35 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 02:33:35 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 02:33:43 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 02:33:43 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:33:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 02:33:51 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357476  bytes 63492550193744 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559780  bytes 161885763499 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:33:59 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-26 02:34:00 - sshInstance.py - [line:54] - root - INFO - 0000:89:00.0

2022-03-26 02:34:08 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:89:00.0 | grep LnkCap:
2022-03-26 02:34:08 - sshInstance.py - [line:54] - root - INFO - 		LnkCap:	Port #8, Speed 2.5GT/s, Width x1, ASPM L0s L1, Exit Latency L0s <64ns, L1 <1us

2022-03-26 02:34:16 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:89:00.0 | grep LnkSta:
2022-03-26 02:34:16 - sshInstance.py - [line:54] - root - INFO - 		LnkSta:	Speed 2.5GT/s (ok), Width x1 (ok)

2022-03-26 02:34:21 - test_nics.py - [line:251] - root - INFO - --check the speed and width of the ethernet enp137s0f0
2022-03-26 02:34:21 - test_nics.py - [line:271] - root - INFO - the linkCap and LnkSta value are correct. The speed and width value of LnkSta are  2.5GT/s and Width x1.
.2022-03-26 02:34:24 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:34:24 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:34:32 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:34:32 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:34:40 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 02:34:40 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:34:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 02:34:48 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:34:56 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 02:34:56 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:01 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 02:35:01 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_PI_Networking_Link_State_Check_02_32_53
2022-03-26 02:35:04 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 02:35:04 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:09 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 02:35:09 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:35:09 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 02:35:09 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Link_State_Check case ============================
2022-03-26 02:35:09 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_ConnectivityShutdown_02_35_09/test_PI_Networking_ConnectivityShutdown.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_ConnectivityShutdown_02_35_09
2022-03-26 02:35:09 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_ConnectivityShutdown case ============================
2022-03-26 02:35:09 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:35:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:35:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 02:35:09 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:35:09 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 02:35:12 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 02:35:12 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 02:35:17 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 02:35:17 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 02:35:20 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 02:35:23 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:28 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 02:35:28 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 02:35:31 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 02:35:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:40 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 02:35:43 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:35:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 02:35:52 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 02:36:00 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 02:36:00 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:36:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 02:36:08 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357495  bytes 63492550197324 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559780  bytes 161885763499 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:36:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11
2022-03-26 02:36:16 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:36:24 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:36:24 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:36:32 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-03-26 02:36:36 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.107 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.034 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.031 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.031/0.048/0.107/0.029 ms

2022-03-26 02:36:41 - power.py - [line:18] - root - INFO - checking the machine status
2022-03-26 02:36:41 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
2022-03-26 02:37:01 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

2022-03-26 02:37:01 - power.py - [line:22] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - test_nics.py - [line:296] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - sshInstance.py - [line:144] - root - INFO - ping sut, please wait ......
2022-03-26 02:38:01 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.691 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.630 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.734 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.792 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.740 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4064ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.630/0.717/0.792/0.059 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:38:06 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:38:06 - power.py - [line:39] - root - INFO - starting to power off the machine....
2022-03-26 02:38:06 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off
....
2022-03-26 02:38:26 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:26 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off

2022-03-26 02:38:26 - power.py - [line:44] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:26 - test_nics.py - [line:299] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:36 - power.py - [line:18] - root - INFO - checking the machine status
2022-03-26 02:38:36 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
2022-03-26 02:38:56 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:56 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

2022-03-26 02:38:56 - power.py - [line:22] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
F2022-03-26 02:38:59 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:38:59 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:07 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-26 02:39:07 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:15 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 0
2022-03-26 02:39:15 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 0
2022-03-26 02:39:23 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:31 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-26 02:39:31 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:36 - test_nics.py - [line:71] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-26 02:39:36 - test_nics.py - [line:73] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-26/test_PI_Networking_ConnectivityShutdown_02_35_09
2022-03-26 02:39:39 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-03-26 02:39:39 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:39:44 - test_nics.py - [line:75] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-26 02:39:44 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-03-26 02:39:44 - test_nics.py - [line:77] - root - INFO - this is Test_Nics teardown
2022-03-26 02:39:44 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_ConnectivityShutdown case ============================
                             [100%]

=================================== FAILURES ===================================
______________ Test_Nics.test_PI_Networking_ConnectivityShutdown _______________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7fd28612d4a8>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicSimple
    def test_PI_Networking_ConnectivityShutdown(self):
        self.bmc_conf = settings.__getattribute__("bmc_conf")
        sut1_ip = "192.168.10.10"
        sut2_ip = "192.168.10.11"
        self.cn_test_sut.ssh_exec_cmd(self.client2, "ifconfig {} {}".format(self.pkg_generator["pf1"], sut2_ip))
        self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {} {}".format(self.sut_conf["pf1"], sut1_ip))
        rs = 0
        cmdip = "ping -c 5 {}\n".format(sut2_ip)
        stdout,stderr = self.cn_test_sut.ssh_exec_cmd(self.client, cmdip)
        if stdout.find("received, 0% packet loss") == -1:
            assert 200 == 400
    
        powers = power()
        output = powers.power_status(self.sut_conf, self.bmc_conf)
        logging.info(output)
        assert 200 == self.cn_test_sut.ping_sut(self.session, self.sut_conf["host"])
        output = powers.power_off(self.sut_conf, self.bmc_conf)
        logging.info(output)
        time.sleep(10)
        output = powers.power_status(self.sut_conf, self.bmc_conf)
>       if output.find("Power is off") > -1:
E       TypeError: a bytes-like object is required, not 'str'

npxtester/Tests/testNics/test_nics.py:302: TypeError
---------------------------- Captured stdout setup -----------------------------
2022-03-26 02:35:09 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_ConnectivityShutdown_02_35_09/test_PI_Networking_ConnectivityShutdown.log, case_logdir: /root/testconfig/testlogs/2022-03-26/test_PI_Networking_ConnectivityShutdown_02_35_09
2022-03-26 02:35:09 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_ConnectivityShutdown case ============================
2022-03-26 02:35:09 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:35:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-26 02:35:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-26 02:35:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-26 02:35:09 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-26 02:35:09 - test_nics.py - [line:31] - root - INFO - --Start rmmod irdma --
2022-03-26 02:35:12 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-03-26 02:35:12 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-03-26 02:35:17 - test_nics.py - [line:33] - root - INFO - --Finish rmmod irdma --
2022-03-26 02:35:17 - test_nics.py - [line:34] - root - INFO - --Start rmmod ice --
2022-03-26 02:35:20 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-03-26 02:35:23 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:28 - test_nics.py - [line:36] - root - INFO - --Finish rmmod ice --
2022-03-26 02:35:28 - test_nics.py - [line:37] - root - INFO - --Start modprobe ice --
2022-03-26 02:35:31 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-03-26 02:35:35 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:40 - test_nics.py - [line:39] - root - INFO - --Finish modprobe ice --
2022-03-26 02:35:43 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 up
2022-03-26 02:35:43 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:35:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i enp137s0f0 | grep ^version:
2022-03-26 02:35:52 - sshInstance.py - [line:54] - root - INFO - version: 1.7.19

2022-03-26 02:36:00 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0
2022-03-26 02:36:00 - sshInstance.py - [line:54] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-26 02:36:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0
2022-03-26 02:36:08 - sshInstance.py - [line:54] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 41946357495  bytes 63492550197324 (57.7 TiB)
        RX errors 0  dropped 212914  overruns 0  frame 0
        TX packets 2231559780  bytes 161885763499 (150.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


----------------------------- Captured stdout call -----------------------------
2022-03-26 02:36:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens801f0 192.168.10.11
2022-03-26 02:36:16 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:36:24 - sshInstance.py - [line:50] - root - INFO - ifconfig enp137s0f0 192.168.10.10
2022-03-26 02:36:24 - sshInstance.py - [line:54] - root - INFO - 
2022-03-26 02:36:32 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-03-26 02:36:36 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.107 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.034 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.031 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.031/0.048/0.107/0.029 ms

2022-03-26 02:36:41 - power.py - [line:18] - root - INFO - checking the machine status
2022-03-26 02:36:41 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
2022-03-26 02:37:01 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

2022-03-26 02:37:01 - power.py - [line:22] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - test_nics.py - [line:296] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:37:01 - sshInstance.py - [line:144] - root - INFO - ping sut, please wait ......
2022-03-26 02:38:01 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.115.71....
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.691 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.630 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.734 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.792 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.740 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - --- 10.239.115.71 ping statistics ---
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 4064ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.630/0.717/0.792/0.059 ms
2022-03-26 02:38:06 - commonFunc.py - [line:30] - root - INFO - 
2022-03-26 02:38:06 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.115.71
2022-03-26 02:38:06 - power.py - [line:39] - root - INFO - starting to power off the machine....
2022-03-26 02:38:06 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off
....
2022-03-26 02:38:26 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:26 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off

2022-03-26 02:38:26 - power.py - [line:44] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:26 - test_nics.py - [line:299] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:36 - power.py - [line:18] - root - INFO - checking the machine status
2022-03-26 02:38:36 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
2022-03-26 02:38:56 - commonFunc.py - [line:25] - root - ERROR - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
2022-03-26 02:38:56 - commonFunc.py - [line:26] - root - INFO - fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

2022-03-26 02:38:56 - power.py - [line:22] - root - INFO - b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 ifconfig ens801f0 192.168.10.11
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig enp137s0f0 192.168.10.10
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.11

INFO     root:sshInstance.py:54 PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.107 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.034 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.035 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.031 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.031/0.048/0.107/0.029 ms

INFO     root:power.py:18 checking the machine status
INFO     root:commonFunc.py:16 starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
ERROR    root:commonFunc.py:25 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:commonFunc.py:26 fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

INFO     root:power.py:22 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:test_nics.py:296 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:sshInstance.py:144 ping sut, please wait ......
INFO     root:commonFunc.py:16 starting to execute the cmd: ping -w 5 10.239.115.71....
INFO     root:commonFunc.py:30 PING 10.239.115.71 (10.239.115.71) 56(84) bytes of data.
INFO     root:commonFunc.py:30 64 bytes from 10.239.115.71: icmp_seq=1 ttl=59 time=0.691 ms
INFO     root:commonFunc.py:30 64 bytes from 10.239.115.71: icmp_seq=2 ttl=59 time=0.630 ms
INFO     root:commonFunc.py:30 64 bytes from 10.239.115.71: icmp_seq=3 ttl=59 time=0.734 ms
INFO     root:commonFunc.py:30 64 bytes from 10.239.115.71: icmp_seq=4 ttl=59 time=0.792 ms
INFO     root:commonFunc.py:30 64 bytes from 10.239.115.71: icmp_seq=5 ttl=59 time=0.740 ms
INFO     root:commonFunc.py:30 
INFO     root:commonFunc.py:30 --- 10.239.115.71 ping statistics ---
INFO     root:commonFunc.py:30 5 packets transmitted, 5 received, 0% packet loss, time 4064ms
INFO     root:commonFunc.py:30 rtt min/avg/max/mdev = 0.630/0.717/0.792/0.059 ms
INFO     root:commonFunc.py:30 
INFO     root:commonFunc.py:31 finished exeucting the cmd: ping -w 5 10.239.115.71
INFO     root:power.py:39 starting to power off the machine....
INFO     root:commonFunc.py:16 starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off
....
ERROR    root:commonFunc.py:25 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:commonFunc.py:26 fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power off

INFO     root:power.py:44 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:test_nics.py:299 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:power.py:18 checking the machine status
INFO     root:commonFunc.py:16 starting to execute the cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status
....
ERROR    root:commonFunc.py:25 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
INFO     root:commonFunc.py:26 fail to executing cmd: ipmitool -I lanplus -H 10.239.182.128 -U root -P Intel@123 power status

INFO     root:power.py:22 b'Error: Unable to establish IPMI v2 / RMCP+ session\n'
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:74
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:74: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:121
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:121: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:396
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:396: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:521
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:521: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:829
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:829: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:84
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:84: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:111
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:111: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:141
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:141: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:212
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:212: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:238
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:238: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:281
  /root/.jenkins/workspace/icx-d-test-nic-simple/npxtester/Tests/testNics/test_nics.py:281: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_simple_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_PI_Networking_ConnectivityShutdown
===== 1 failed, 7 passed, 9 deselected, 15 warnings in 3683.76s (1:01:23) ======
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
Stage "collect logs" skipped due to earlier failure(s)
[8mha:////4AfI3SthhIf/wHKBxkZZVuznCBOLGICd+mTaK5J1XhvpAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMHmfoKzrT+3fkMfZ274AAAA=[0m[Pipeline] }
[8mha:////4O1G14z5oV6C9jl1Q5yIn+pZFQRmsElhyj6JiXyHA8ixAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiBB2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwhQ7nbf2r7BpKm1V++AAAA[0m[Pipeline] // stage
[8mha:////4KtBCOpIKS64HlRGbxVDHBnhT4546C6ceKSEgLxKWO63AAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUauVY2WZoPEGEGAOZXUwWofJEXs07yMiMlb/67zXv9cYyRRw5OtVYaj2lyqsu9G56auDYXgMPquGLqpgSB6tKO5Rc29OMJYvFvCzHQmNlqQqcPDnBWjfmYYpgyBVniZM7aOS+vuOJTE9lMVG+MEZsbn2dmH6dvwGMXSfIt3tBtvsA/gVtV70AAAA=[0m[Pipeline] }
[8mha:////4PVa+eOfGCB5ryE2GyokIfyPZOVvh3emQVgJTOuHuCGAAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IMJC7GybVh8Qsq1Fpo3sP2IUx+kb/mP0gkcfJO95zlvN5Yp4gjR6daS52nVHvVh8HNT40cu2vgUbV8UTVT4mBVZceKG3tasGKxWJblWGlsLNWBkycn2OrWPEwRDLniLHF2B43cN3c8kem5LCbKF6aI3W1oEtOv8zeAqe8FebkXZOUHjeqpjr0AAAA=[0m[Pipeline] // node
[8mha:////4OyM7hMK6utvbTfbnNkjhtaoCHWCq3+ZbQ+BNe9AvJHZAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKRoKBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Lu9INt+AKvzzsS9AAAA[0m[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
